source/fundamentals/indexes.txt

```diff
@@ -128,23 +128,22 @@ To learn more, see  :manual:`Compound Indexes </core/index-compound>`.
 Multikey Indexes (Indexes on Array Fields)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Multikey indexes** are indexes that improve performance for queries that
-specify ascending or descending indexes on fields that contain an array value.
-You can define a multikey index using the same syntax as a single field or
-compound index.
+**Multikey indexes** are indexes that improve performance for queries on
+fields that contain array values. You can define a multikey index using
+the same syntax as when you create a single field or compound index.
 
-The following example use the ``createIndex()`` method to create an ascending
-index on the ``cast`` field (array of names) in the ``movies`` collection in
-the ``sample_mflix`` database.
+The following example uses the ``createIndex()`` method to create an ascending
+index on the ``cast`` field, which contains an array of names, in the
+``movies`` collection of the ``sample_mflix`` database.
 
 .. literalinclude:: /code-snippets/indexes/multikey.js
    :language: js
    :start-after: begin-idx
    :end-before: end-idx
    :dedent:
 
-The following is an example of a query that would be covered by the index
-created above.
+The following code performs a query that uses the index on
+the ``cast`` field:
```

Comment:
Suggestion:
I think the previous wording is more precise than "uses", but here's an alternative that is less wordy that could work:

```suggestion
The following code queries the multikey index on the ``cast`` field
```